### DISCUSSION TITLE
[Question] How to add this into GSA CaptchaBreaker

### DISCUSSION THREAD

**Discussion by rastarr** [QIN2DIM/hcaptcha-challenger#786] [Ideas]
GSA Captcha Breaker is a great captcha solver which is missing Hcaptcha and Recaptcha
Has anyone found a way to integrate this project and the companion Recaptcha project into GSA Captcha Breaker?

**Comment by ItsCEED** [comment]
GSA is fairly closed sourced, and it will never work with something like this as hcap in general needs a lot more than just OCR.

**Comment by rastarr** [comment]
> GSA is fairly closed sourced, and it will never work with something like this as hcap in general needs a lot more than just OCR.

The thing is the GSA CB can send to other captcha services which is why I've asked

![CleanShot 2023-01-21 at 7 53 48](https://user-images.githubusercontent.com/2160922/213830489-9a89af53-effd-4aa0-80ed-1272c14c10cc.jpg)

**Comment by ItsCEED** [comment]
I know, GSA user my self.
But personally i dont see a use case for to implement this in GSA

**Comment by rastarr** [comment]
> I know, GSA user my self. But personally i dont see a use case for to implement this in GSA

So people can use GSA CB to solve Hcaptcha and Recaptcha instead of paid services. That's my use case anyway.

**Comment by QIN2DIM** [comment]
@rastarr It sounds good, how does it work exactly －O－
